### PR TITLE
Add a bypass permission for towny protections

### PR DESCRIPTION
**Pull request by partydev**

### CODE REVIEW

File: src/main/java/io/github/rypofalem/armorstandeditor/PlayerEditor.java [hunk 1]
```diff
@@ -115,8 +115,8 @@ public void editArmorStand(ArmorStand armorStand) {
 
 		//FIX for https://github.com/Wolfst0rm/ArmorStandEditor-Issues/issues/15
 		if (plugin.getServer().getPluginManager().getPlugin("Towny") != null) {
-			if (TownyAPI.getInstance().isWilderness(getPlayer().getLocation())) return;
-			if (!TownyActionEventExecutor.canDestroy(getPlayer(), getPlayer().getLocation().getBlock().getLocation(), Material.ARMOR_STAND)) return;
+			if (TownyAPI.getInstance().isWilderness(getPlayer().getLocation()) && !getPlayer().hasPermission("asedit.bypass")) return;
```
Comment (Wolfieheart) [inline src/main/java/io/github/rypofalem/armorstandeditor/PlayerEditor.java:118]: Could it be possible that this already fixed in Development? 

See this for more info: https://github.com/Wolfieheart/ArmorStandEditor/blob/dev/1.18.2-35/src/main/java/io/github/rypofalem/armorstandeditor/protections/TownyProtection.java

**Review by Wolfieheart — CHANGES_REQUESTED** [review]
Something that might need to be checked before I merge this in. I appreciate the contribution however, just something to be sure of before we go further.

**Comment by partydev** [comment]
Understood, I did not notice that your dev branch addressed this. It was an issue that I noticed and fixed for my server and I wanted to pay it forward. Will close this and will check that branch in case I come up with any more ideas.

**Comment by Wolfieheart** [comment]
No worries. Thank you for contributing in any case and please do so in the
future if you think there are ideas in Dev that could be fixed in the long
run.

**Pull request closed without merging**